tests/two-way-iframe.comlink.test.js

```diff
@@ -24,17 +28,37 @@ describe("Comlink across iframes", function () {
 
   afterEach(function () {
     this.ifr.remove();
+    chai.config.truncateThreshold = oldTruncateThreshold;
   });
 
   it("can communicate both ways", async function () {
-    let called = false;
+    let notcalled = true;
+    let error = "";
+    window.addEventListener("unhandledrejection", (ev) => {
```

Comment:
That's how we get notified in this test that Comlink try to do a `postMessage` (from the `Comlink.expose` event listener) that it shouldn't (but it does in v4.3.0 because of the overlap between `WireValueType` and `MessageType`).

In order to get an exception, we're forcing here Comlink to do a `postMessage` on an object that can't be cloned.